src/components/tag/tag.classnames.ts

```diff
@@ -39,14 +43,31 @@ export const TagSizes: Record<TagSize, string> = {
   lg: 'px-800 py-600 text-100 gap-x-800',
 }
 
-export const TagIcons: Record<
-  TagType,
-  React.ForwardRefExoticComponent<Omit<QuillSvgProps, 'ref'>>
-> = {
-  error: LabelPairedTriangleExclamationBoldIcon,
-  warning: LabelPairedCircleExclamationBoldIcon,
-  success: LabelPairedCircleCheckBoldIcon,
-  info: LabelPairedCircleInfoBoldIcon,
+type TagIcons = {
+  [key in TagType]: {
+    [key in 'true' | 'false']: React.ForwardRefExoticComponent<
```

Comment:
```
    [key in TagProps['bold']]: React.ForwardRefExoticComponent<
```
Not sure if this is the right way of writing it but when I wrote the above snippet it gave me errors. Please advise on this aswell 😅

Reply:
Can we use `bold` and `regular` strings instead of `true` or `false` ?

Reply:
@prince-deriv I think having `bold` with `true` or `false` is much more straightforward and more natural don't you think?

Reply:
please stick with `true` and `false` on these kind of stuff.